pom.xml

```diff
@@ -4,14 +4,27 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.596.1</version>
+        <version>3.50</version>
         <relativePath />
     </parent>
     <artifactId>liquibase-runner</artifactId>
     <version>1.4.0-SNAPSHOT</version>
     <name>Liquibase Runner</name>
 
     <packaging>hpi</packaging>
+    <properties>
+        <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
+        <jenkins.version>2.223</jenkins.version>
+        <java.level>8</java.level>
+        <!-- Other properties you may want to use:
+          ~ jenkins-test-harness.version: Jenkins Test Harness version you use to test the plugin. For Jenkins version >= 1.580.1 use JTH 2.0 or higher.
+          ~ hpi-plugin.version: The HPI Maven Plugin version used by the plugin..
+          ~ stapler-plugin.version: The Stapler Maven plugin version required by the plugin.
+     -->
+     	<!-- 
+        <workflow.version>2.6</workflow.version>
+         -->
+    </properties>
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Liquibase+Runner</url>
```

Comment:
If you want to update the documentation, you will need to migrate it to GitHub. Can be done in a separate pull request.
https://jenkins.io/blog/2019/10/21/plugin-docs-on-github/